QUARKUS-728.md

```diff
@@ -0,0 +1,24 @@
+# Quarkus-728 - Add support for OpenJDK 17
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-728
+
+The goal is to verify OpenJDK 17 integration
+
+## Scope of the testing
+Test development will focus on  
+ - New specifict scenarios will be created to cover the integration of new objects as “sealed” or “records” classes with several extensions as Hibernate or RestEasy.
+ - Current escenarios must work with Java 17  
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works both in JVM and NATIVE mode 
+ - Ensure this coverage works in Openshift (s2i)
+ - New test suite will be created in order to develop specifict Java 17 scenarios
+ - New test suite will use Gradle as a build tool
+    
```

Comment:
No information about impact on 
 - jobs definition
 - test execution time
 - lab capacity
 - test cycle length

Will all the Java 11 jobs have Java 17 variants?

Where will be the Java installation taken from, is RHEL going to support it via RPMs? How will be the tooling for VMs preparation changed to cover new JDK on Jenkins slaves, Jenkins master tools config will need some small Java 17 related adjustments too.

Reply:
@mjurc could you help me to clarify which Jobs should support Java17?

Reply:
Please check Testing strategy document for RHBQ 2.x, it speaks about strategy for platforms, the idea is to have primary one and subset of coverage for other platforms.

The same applies to Java 11 and Java 17. Java 11 should be the primary version, subset of coverage for Java 17 and over time (assume 1-2 years) we switch Java 17 as the primary platform and Java 11 as the subsequent one.

For Java 17 we need to execute the jobs with the new TS for sure + Java 17 specific TS + Quickstarts (with redefined source / target to 17 via command line). Maybe also StartStopTS should be executed with Java 17.

Reply:
I have re-write all of this section following your comments.